acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/webdriver/WebDriverProvider.java

```diff
@@ -85,6 +85,8 @@ public void initialise() {
         desiredCapabilities.setCapability(ChromeOptions.CAPABILITY, chromeOptions);
 
         webDriver = new RemoteWebDriver(webDriverServiceProvider.getUrl(), desiredCapabilities);
+
+        webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```

Comment:
What does this do?

Reply:
As discussed offline, this is a way to globally apply default timeouts to WebDriver 'find' operations. It appeared to have helped with some of the tests but, later attempts with this statement removed proved that it's not strictly necessary thus, it makes sense to remove it as it incurs a risk of unnecessarily inflating test run time. The statement is now gone.